nifi-stateless/nifi-stateless-bundle/nifi-stateless-engine/src/main/java/org/apache/nifi/stateless/core/RegistryUtil.java

```diff
@@ -119,6 +126,10 @@ public VersionedFlowSnapshot getFlowContents(final String bucketId, final String
         return flowSnapshot;
     }
 
+    public String getBaseRegistryUrl(String storageLocation) throws MalformedURLException {
```

Comment:
As this appears to be exposed only for testing, the visibility can be reduced to `protected`.
```suggestion
    protected String getBaseRegistryUrl(final String storageLocation) {
```